setup.py

```diff
@@ -102,6 +102,7 @@ def check_setuptools_features():
         'logstats==0.2.1',
         'base58==0.2.2',
         'flask==0.10.1',
+        'flask-restful==0.3.5',
```

Comment:
I think we can safely use: `flask-restful~=0.3.0` in this way we pin the `0.3.x` version of `flask-restful`. They should use semantic versioning. (Yes, we should apply the same pattern to the other requirements, we can do it later).